### PR TITLE
add raft_boltdb config key

### PR DESCRIPTION
**Pull request by chuckyz**
This adds raft_boltdb for the upcoming 1.11 release.

### CODE REVIEW

**Comment by Vulfox** [comment]
LGTM: Version bump in metadata.rb please :)

**Comment by chuckyz** [comment]
@Vulfox ping

**Review by Vulfox — APPROVED** [review]

File: libraries/consul_config.rb [hunk 1]
```diff
@@ -210,6 +211,7 @@ def params_to_json
           protocol
           reap
           read_replica
+	  raft_boltdb
```
Comment (roberteckert) [inline libraries/consul_config.rb]: not to pick nits but is this spacing correct

Comment (chuckyz) [reply libraries/consul_config.rb]: if ruby doesn't care i don't care

Comment (chuckyz) [reply libraries/consul_config.rb]: you care though so i fixed it ❤️

**Pull request merged**